index.html

```diff
@@ -290,6 +293,33 @@ <h3>Contact me</h3>
         </form>
       </div>
     </section>
-    <script src="script.js"></script>
+    
+    <!-- popup code starts -->
```

Comment:
- [ ] As the previous reviewer mentioned your popup should look like the Figma design as much as possible, kindly consider centering the popup and adding a blurred background to it

|Your design|Figma design|
|-|-|
|![Screenshot_20220729_054229](https://user-images.githubusercontent.com/25227022/181685480-22e80cae-7119-4b7f-8971-36d3e62e2ce9.png)|![](https://user-images.githubusercontent.com/66230306/181614032-bc3136c1-8a79-49b4-a772-6256fe896829.png)|